app/src/main/AndroidManifest.xml

```diff
@@ -69,7 +69,6 @@
         </activity>
         <activity
             android:name=".views.LoginActivity"
-            android:label="Sign In"
```

Comment:
It doesn't reappear in the string file ?

Reply:
@teolemon The string is equivalent to "R.string.txtSignIn" which is translatable . Hence label has been removed and title has been set using "R.string.txtSignIn" in LoginActivity.

Reply:
@teolemon 
<img width="753" alt="cat2" src="https://user-images.githubusercontent.com/30733262/36975659-bf6f5f76-20a0-11e8-9e9a-b5e085eed59d.png">
<img width="1114" alt="cat" src="https://user-images.githubusercontent.com/30733262/36975663-c3696644-20a0-11e8-8733-b7bd7ba434f1.png">

The title of category activity is set in the activity itself. There is no label for category activity in manifest.xml . 

My changes are similar to the existing code in category activity .

Reply:
This pull request is solved .